test/integration/targets/vmware_guest_powerstate/tasks/poweroff_d1_c1_f0.yml

```diff
@@ -1,52 +0,0 @@
-# Test code for the vmware_guest_powerstate module.
-# Copyright: (c) 2017, Abhijeet Kasurde <akasurde@redhat.com>
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
-
-- name: Wait for Flask controller to come up online
```

Comment:
Not sure why these are removed ?

Reply:
I merged them in a single main.yml file.